recipe/meta.yaml

```diff
@@ -125,6 +128,19 @@ outputs:
       commands:
         - python -c "import ray; ray.init()"
 
+  - name: ray-default
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('ray-core', exact=True) }}
+        - colorful
+
+    test:
+      imports:
+        - ray
```

Comment:
You should still add a comment when the test import does not match the output name (i.e. `ray-default` here). Something like "ray-default is a convenience wrapper with no imports of its own"

Reply:
Sounds good, added!